### PR TITLE
Feature: Migrate client to Next.js in TypeScript

### PR DESCRIPTION
**Pull request by phanlyhuynh**
image

### CODE REVIEW

**Comment by phanlyhuynh** [comment]
If it's good, please adding the "hacktoberfest-accepted" label. Thanks!!!

**Review by sidsbrmnn — CHANGES_REQUESTED** [review]
Thanks for the contribution! There are a few changes needed here. Could you remove the react-scripts from dependencies as it's not needed in a Next project and also remove yarn.lock as I want to maintain it using package-lock.json. Once these changes are made, the PR is good to go.

File: package.json [hunk 1]
```diff
@@ -5,17 +5,17 @@
...
     "axios": "^0.19.0",
     "bootstrap": "^4.3.1",
-    "jquery": "^3.4.1",
+    "jquery": "^3.5.1",
+    "next": "^9.5.4",
     "popper.js": "^1.15.0",
     "react": "^16.9.0",
     "react-dom": "^16.9.0",
-    "react-scripts": "3.1.1"
+    "react-scripts": "^3.4.3"
```
Comment (sidsbrmnn) [inline package.json:13]: react-scripts are not needed.

**Review by sidsbrmnn — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (phanlyhuynh) [reply package.json]: @sidsbrmnn it's done !!!

**Comment by phanlyhuynh** [comment]
@sidsbrmnn Please check for me. Thanks !!!

**Comment by sidsbrmnn** [comment]
Perfect! Thanks for contributing

**Comment by phanlyhuynh** [comment]
@sidsbrmnn why were you add "invalid" label?

**Comment by sidsbrmnn** [comment]
@phanlyhuynh the app crashed completely with wrong implementation of state management in functional components.

**Pull request merged**